src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<Fruit, Integer> fruits = new HashMap<>();
```

Comment:
According to the encapsulation principle, it's recommended to restrict the access to class fields. In this case, the `fruits` map should be private and accessed via public methods. This will prevent direct modification from outside the class, which can lead to hard-to-track bugs and inconsistencies in the data.